src/arguments/CoreBoolean.ts

```diff
@@ -9,12 +9,13 @@ export class CoreArgument extends Argument<boolean> {
 
 	public run(parameter: string, context: { readonly truths?: string[]; falses?: readonly string[] } & Argument.Context): Argument.Result<boolean> {
 		const resolved = resolveBoolean(parameter, { truths: context.truths, falses: context.falses });
-		if (resolved.isOk()) return this.ok(resolved.unwrap());
-		return this.error({
-			parameter,
-			identifier: resolved.unwrapErr(),
-			message: 'The argument did not resolve to a boolean.',
-			context
-		});
+		return resolved.mapErr((identifier) =>
```

Comment:
`mapErrInto`? Was this not possible?

https://github.com/sapphiredev/framework/pull/468#discussion_r932544618

Reply:
I didn't see that edit, I don't see any method called `mapErrInto` and I don't see another comment mentioning it. I'm not sure what that is.

Reply:
https://github.com/sapphiredev/utilities/pull/404/files#diff-63b98d6beb14a27a657df5479d7a87f5136e762688cc29c9f4543a96ac8a492aR64

Reply:
@samfundev Wait for next result and update to that.
https://github.com/sapphiredev/utilities/pull/404/files#diff-63b98d6beb14a27a657df5479d7a87f5136e762688cc29c9f4543a96ac8a492aR64